src/paper/management/commands/add_tags.py

```diff
@@ -0,0 +1,49 @@
+'''
+Add tags for all papers using the abstract
+'''
+from django.core.management.base import BaseCommand
+
+from paper.models import Paper
+from tag.models import Tag
+
+import nltk
+from nltk.corpus import stopwords
+import yake
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        papers = Paper.objects.filter(abstract__isnull=False)
+        print('Adding tags to papers')
+        nltk.download('stopwords')
+        nltk.download('punkt')
+        for paper in papers:
+            keyword_settings = {
+                "lan": "en",
+                "n": 1,
+                "dedupLim": 0.1,
+                "top": 3,
+                "features": None
+            }
+
+            custom_kw_extractor = yake.KeywordExtractor(**keyword_settings)
+
+            abstract = paper.abstract.lower() + paper.title.lower()
```

Comment:
Maybe you already considered it but I think `paper_title` property is worth while checking out. It is different than title in many cases - one refers to the "frontend" title and the other to the "official" paper title